frontend/src/App.tsx

```diff
@@ -57,6 +58,7 @@ export const App: React.FC<AppProps> = ({ userConfig, appConfig }) => {
   const [viewState, setViewState] = useAtom(viewStateAtom);
   const [filename, setFilename] = useState(getFilenameFromDOM());
   const [savedCodes, setSavedCodes] = useState<string[]>([""]);
+  const [savedConfigs, setSavedConfigs] = useState<CellConfig[]>([]);
```

Comment:
in a followup, for QOL, i'll try to move this stale state to the reducer (less logic in the components)